orientdb/src/main/scala/akka/stream/alpakka/orientdb/javadsl/OrientDBFlow.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.orientdb.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.orientdb._
+import akka.stream.javadsl.Flow
+import com.orientechnologies.orient.core.record.impl.ODocument
+
+import scala.collection.JavaConverters._
+import java.util.{List => JavaList}
+
+object OrientDBFlow {
+
+  /**
+   * Java API: creates a [[OrientDBFlowStage]] that accepts as ODocument
+   */
+  def create(
+      className: String,
+      settings: OrientDBSinkSettings
+  ): Flow[IncomingMessage[ODocument], JavaList[IncomingMessage[ODocument]], NotUsed] =
+    akka.stream.scaladsl.Flow
+      .fromGraph(
+        new OrientDBFlowStage[ODocument, JavaList[IncomingMessage[ODocument]]](
+          className,
+          settings,
+          _.asJava,
+          None
+        )
+      )
+      .mapAsync(1)(identity)
```

Comment:
What do you achieve with this?

Reply:
This just resolves the future.
Like apply returns `Flow[IncomingMessage[ODocument], Seq[IncomingMessage[ODocument]], NotUsed]` but if the future is not resolved, then it would return `Flow[IncomingMessage[ODocument], Future[Seq[IncomingMessage[ODocument]]], NotUsed]`

Reply:
Ok